src/plugins/cordova-plugin-geolocation/sim-host.js

```diff
@@ -487,6 +487,13 @@ module.exports = function (messages) {
                 }
             });
 
+            var navSimulationLabel = document.querySelector('[data-loc-id="f89bb5e3"]');
+            var navSimulationTitle = navSimulationLabel && navSimulationLabel.value;
+            if (navSimulationTitle) {
+                gpxFileButton.spoken = [navSimulationTitle, gpxFileButton.textContent].join(' ');
+                gpxGo.spoken = [navSimulationTitle, gpxGo.textContent].join(' ');
+            }
```

Comment:
Rather than this, simply set the `spoken-text` attribute of the to the full value. This is better for localization (while it may be correct in some languages to simply join the two strings, it may not be correct in other languages).